lib/main.js

```diff
@@ -7,14 +7,14 @@
 
     var _ = require('lodash');
 
-    var Main = function(apiKey) {
+    var Main = function(apiUrl, apiKey) {
```

Comment:
Swap the position, existing users only pass `apiKey`, therefore, it should become:

```javascript
var Main = function(apiKey, apiUrl) {
```